gtk2_ardour/foldback_strip.cc

```diff
@@ -855,6 +855,8 @@ FoldbackStrip::build_route_select_menu ()
 bool
```

Comment:
The hash in the commit message is not found. If it is a reference to a commit that hasn't landed yet, it will probably be rebased and the hash will change. Perhaps better to repeat the relevant parts of the commit message.

Reply:
It should be 8c07b1726acb65dbafa329c75eedb5a4d1b1efb8, my mistake.